packages/core/src/common/quick-pick-service.ts

```diff
@@ -14,6 +14,9 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 import { Event } from './event';
+
+// TODO: fix the import.
+// eslint-disable-next-line @theia/runtime-import-check
```

Comment:
Rather than adding these `TODO` annotations and then turning off the linter, might it make sense to set the level for the rule to `warning` for now, make a PR (or several) to fix these, and then switch it to `error`?

Reply:
The idea is to shoot down any new code that would mis-import code, and to do this we really need it to be an error. It also doesn't help that we already swim in a sea of warnings, so new bogus imports would get lost in the noise...

I think we can just open a followup issue inviting whoever will look at those odd imports to search for this comment.

Reply:
We're also swimming in a sea of `TODO` comments, but I take your meaning :-). It depends on how quickly we _actually_ intend to do the todo's. If it's something we intend to address soon, then I don't think we need them. If it's not something we intend to address soon, then it doesn't really sound like a `TODO` at all, and anyone interested in addressing them could search for the disable comments.